### PR TITLE
Always convert `full` to jpeg and use it in manifest thumbnail

### PR DESCRIPTION
**Pull request by atomotic**
This is my attempt to fix the conversion of `full` tiles to jpeg. 
I am still testing on my tropy collections

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -187,6 +187,7 @@ class TropiiifyPlugin {
       await new Promise((resolve, reject) => {
         sharpInstance
           .resize(maxDimension, maxDimension, { fit: 'inside' })
+          .jpeg()
```
Comment (martimpassos) [inline src/plugin.js]: Hi @atomotic, you will need to change the destination path in line 205 as well so the file gets saved with the correct extension

**Comment by martimpassos** [comment]
Thanks for this!

File: src/plugin.js (hunk 1 above)
Comment (atomotic) [reply src/plugin.js]: yes, it's in this commit https://github.com/arkalab/tropiiify/pull/21/commits/3286664277ba01d8e6e0720778e18aeb49dc85ff sorry if they are different, I went ahead by trial and error

**Comment by martimpassos** [comment]
solves #20

**Pull request merged**